app/components/Input/ApprovableInput.js

```diff
@@ -236,14 +236,7 @@ const ApprovableInput = ({
             showMonthDropdown
             showYearDropdown
             calendarClassName="schedule-date-picker"
-            popperModifiers={{
-              preventOverflow: {
-                padding: 10,
-              },
-            }}
-            popperProps={{
-              positionFixed: true,
-            }}
+            popperPlacement="bottom-start"
```

Comment:
Shouldn't you do the same change in `app/components/FormikDatePicker/index.tsx`?